Update prev Project setup page and links to content (#6690)
aws-amplify/docs: src/pages/[platform]/start/project-setup/prerequisites/index.mdx

  };
}

<Callout info>

> Project setup guides are available for the [previous version](/[platform]/prev/start/project-setup/prerequisites/). However, it is highly recommended that you use the latest version to set up new projects following this guide.

Reviewer: Change second sentence to "However, we highly recommend using the latest version and following this guide when setting up new projects."

> Project setup guides are available for the [previous version](/[platform]/prev/start/project-setup/prerequisites/). However, it is highly recommended that you use the latest version to set up new projects following this guide.

Author: ```suggestion
Project setup guides are available for the [previous version](/[platform]/prev/start/project-setup/prerequisites/). However, we highly recommend using the latest version and following this guide when setting up new projects.
```
